docs/includes/swagger/cds_full.json

```diff
@@ -2482,7 +2482,7 @@
             "type" : "array",
             "description" : "Fully described deposit rates for this account based on the equivalent structure in Product Reference",
             "items" : {
-              "$ref" : "#/definitions/BankingProductLendingRate"
+              "$ref" : "#/definitions/BankingProductLendingRateV2"
```

Comment:
Incorrect version number detected. Should be V3 with backfill.
https://github.com/ConsumerDataStandardsAustralia/standards-maintenance/issues/311